tagger.py

```diff
@@ -0,0 +1,66 @@
+import django
+import os
+
+# Need to set up django app first
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings')
+django.setup()
+
+# ---
+
+import logging
+import sys
+from datetime import datetime
+
+from data_processors import services
+from utils import libs3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+log_file = logging.FileHandler("tagger-{}.log".format(datetime.now().strftime("%Y%m%d%H%M%S")))
+log_file.setLevel(logging.INFO)
+log_file.setFormatter(logging.Formatter("%(asctime)s %(name)-12s %(levelname)-8s %(message)s"))
+logger.addHandler(log_file)
+
+
+FILTER_SUFFIX = ".bam"
+
+if __name__ == '__main__':
+    """
+    Meant to run one time, offline tool to re/process tagging as implemented in services.tag_s3_object(..)
+    
+    Cost:
+    1. Listing bucket is paginated 1000 objects per request, see libs3.get_matching_s3_keys(..), therefore
+        450,000 objects * $0.0055 / 1000 = $2.475
+    2. Tagging an object requires 1 GET request and 1 PUT request, therefore, say 2712 .bam out of 450,000 
+        (2712 * $0.0055) PUT + (2712 * $0.00044) GET = $14.916 + $1.19328 = $16.10928
```

Comment:
Actually:
450,000 objects at 1000 objects per request = 450 LIST requests
450 requests at $0.0055 per 1000 requests = $0.002475

Similarly:
1 PUT + 1 GET request at $0.0055 and $0.0044 per 1000 requests = $0.00594 per 1000 requests
For 2712 objects thats 2712 * 0.00594 / 1000 = 0.01610928

negligible cost in any case ;-)

Reply:
Thanks Florian. Updated the note.